### ISSUE TITLE
Attempting to drag metabox into the sidebar causes the page to expand beyond itself

### ISSUE THREAD

**Issue by garrett-eclipse** [WordPress/gutenberg#17432]
**Describe the bug**
If you attempt to move a metabox from the main content section into the sidebar the entire editor starts expanding making it really hard to get to the sidebar (see screengrab).

**To reproduce**
1. Go to the block editor with a metabox enabled like from Co-Authors Plus in my example.
2. Attempt to drag the metabox to the sidebar.
3. Note how hard it is to get to the sidebar as the entire page is starting to expand beyond it's bounds.

**Expected behavior**
The editor shouldn't expand as we drag the metabox it should stay locked making it easier to get your metabox into the sidebar.

**Screenshots**
![facdb6e1f0f07362e78342d216e47c42](https://user-images.githubusercontent.com/8726005/64884780-5d55b280-d617-11e9-8d62-8d09c4a0ae8b.gif)

Desktop (please complete the following information):

OS: Mac 10.14.5
Browser: Chrome 76.0.3809.132
Additional context

Gutenberg Core for WordPress 5.2.3

**Comment by draganescu** [comment]
Hey there @garretthyder with WordPress 5.4 and Gutenberg 7.9.1 this seems to be somewhat fixed:

![drag-custom-fields](https://user-images.githubusercontent.com/107534/80127362-e1541d80-859c-11ea-9dd5-486551dda3e8.gif)

Also, in no way was I able to move a metabox to the sidebar :)

**Comment by garretthyder** [comment]
Thanks for testing @draganescu. From your screenshot and retesting just now it seems things have gotten worse.
Now you're unable to move a metabox to the sidebar and dragging off to the right seems to go infinitely.

It would be nice to be able to put metaboxes in the sidebar as was available for the classic editor and the block editor before now. As well would be nice if the frame was locked so dragging off to the right doesn't take the user into the great beyond.

**Comment by jg314** [comment]
I can confirm this is still a bug in WordPress 5.5. While testing the [Wired Impact Volunteer Management](https://wordpress.org/plugins/wired-impact-volunteer-management/) plugin I dragged a sidebar metabox into the main content area. Instead of being able to drag it back, the main content area drags over to the right indefinitely. This seems to match what others have described here.

**Comment by carolinan** [comment]
Confirmed that this is still an issue in 6.0.1 with and without Gutenberg  13.7.0-rc.1 .
I did not use a third party plugin, but the default custom fields meta box.

**Comment by carolinan** [comment]
Similarly I can drag a meta box while it is inside the sidebar, but I can not drag it from the sidebar to the main area.

**Comment by ghost** [comment]
How can we move this forward? It's been 4 years, can we get some more attention in this one by mentioning it in the meetings? I have no clue where to even start if I'd try to fix this.

**Comment by draganescu** [comment]
> From your screenshot and retesting just now it seems things have gotten worse.

Yes @garretthyder , I wasn't aware one _could_ move a metabox to the sidebar in the 1st place 🤦🏻 

> How can we move this forward?

The usual ways, you said "mentioning it in the meetings" which is a good step. It'd be great if say a plugin author who depends on metaboxes could try a PR? Or it will end up being worked on by core contributors at some point.

> It's been 4 years

It's true. All large open source projects suffer from this problem - and dare I say all large software projects in general: longstanding issues which have small impact therefore are never prioritized enough to get to the development pipeline. WordPress in particular outside of Gutenberg has issues fifteen years old in Trac 😄  It's a given.

**Comment by ghost** [comment]
> All large open source projects suffer from this problem

I know, not mentioning it as critique, just reasoning it would be nice to resurface this one :)
